Packs/Kenna/Integrations/KennaV2/KennaV2.py

```diff
@@ -399,8 +399,8 @@ def update_asset_command(client: Client, args: dict[str, str]) -> CommandResults
 
     result = client.http_request(message='PUT', suffix=url_suffix, data=asset)
     if result.get('status') != "success":
-        return CommandResults(readable_output=f'Could not update asset with ID {asset_id}.', raw_response=result)
-    return CommandResults(readable_output=f'Asset with ID {asset_id} was successfully updated.')
+        return f'Could not update asset with ID {asset_id}.',{},[]
+    return f'Asset with ID {asset_id} was successfully updated.',{},[]
```

Comment:
In order to get the same behavior as before using **CommandResults**, you should do the following:
```suggestion
    return CommandResults(readable_output= readable_output, raw_response=readable_output)
```